### DISCUSSION TITLE
renderToString deprecated?

### DISCUSSION THREAD

**Discussion by the-new-kai-lu** [wojtekmaj/react-pdf#1474] [General]
WebStorm shows renderToString as deprecated - is this true? I can't find any documentation to this effect. If it's true, is there something we should be using instead?

**Comment by wojtekmaj** [comment]
I don't understand, is that a generic React question?

The only place where that could be useful in React-PDF that comes to mind is customTextRenderer. Theoretically, renderToString is meant for using React on the server side, and indeed there are methods that make the server more performant and supporting e.g. Suspense. I don't think it's relevant to customTextRenderer.

**Comment by the-new-kai-lu** [comment]
So this page: https://react-pdf.org/node#rendertostring has documentation on "renderToString", and it doesn't indicate that it's deprecated, but WebStorm seems to think this is deprecated?

**Reply by wojtekmaj** [reply]
You seem to be confusing react-pdf with @react-pdf/renderer. This repo has nothing to do with the link you posted.
